common/src/services/api.service.ts

```diff
@@ -225,7 +226,7 @@ export class ApiService implements ApiServiceAbstraction {
     const identityToken =
       request instanceof ApiTokenRequest
         ? request.toIdentityToken()
-        : request.toIdentityToken(this.platformUtilsService.identityClientId);
+        : request.toIdentityToken(this.platformUtilsService.getClientType());
```

Comment:
Update `toIdentityToken` to expect either values in `ClienType` enum or a `ClientType` enum itself